src/api/guardian.js

```diff
@@ -0,0 +1,15 @@
+const Guardian = require("guardian-js");
+const guardian_key = process.env.GUARDIAN_KEY;
+const guardianKey = (query, cb) => {
+  const api = new Guardian(guardian_key, false);
```

Comment:
it's called api that's not good for a variable name

Reply:
give me a proper name to call it

Reply:
it's your code not mine i not supposed to tell you what you should call the variables, my job is to point out if they are bad.

 it can be as easy as adding the name **guardian**  `guardianAPI` <-- this makes more sense

Reply:
also read this https://github.com/nervnews/nerv/pull/21#pullrequestreview-88809001

Reply:
It's not okay to call it api inside the "guardian.js" file but it's okay to call it guardianAPI